.swiftlint.yml

```
@@ -1,3 +1,38 @@
+custom_rules:
+  missing_swifterswift_prefix:
+    included: ".*\\.swift"
+    excluded: ".*Tests\\.swift"
+    name: "Missing SwifterSwift: prefix"
+    regex: "///\\s(?!SwifterSwift:\\s)\\w+[^\\n]*$"
+    match_kinds:
+      - doccomment
+    message: "Doc string should always start with the SwifterSwift: prefix"
+    severity: warning
+  unnecessary_type:
+    name: "Unnecessary Type"
+    regex: '(?sm)[ \ta-zA-Z0-9]?(?:let|var){1} [ \ta-zA-Z0-9]+?:[ \t]+?([a-zA-Z0-9]+?)[\t ]+?=[\t ]?\1'
+    message: "Type Definition Not Needed"
+    severity: warning
+  comments_space:
+    name: "Space After Comment"
+    regex: '(//\w+|///\w+)'
+    message: "There should be a space between the start of a comment and the first character."
+    severity: warning
+    match_kinds:
+      - comment
+      - doccomment
+      - doccomment.field
+  empty_commented_line:
+    name: "Empty Commented Out Line"
+    regex: '(\t+| +)//\n'
```

Again, can you check for whitespace chars? Or will this match newlines too?
```suggestion
    regex: '(\s+)//\n'
```
If `\s` matches newlines we can also do
```suggestion
    regex: '^(\s+)//$'
```

---

1. No, doing so will match lines like `//  SwifterSwift`
2. Yes, `$` looks better here